models/user/user.go

```diff
@@ -443,6 +443,17 @@ func (u *User) GetDisplayName() string {
 	return u.Name
 }
 
+// GetCompleteName returns the the full name and username in the form of
+// "Full Name (@username)" if full name is not empty, otherwise it returns
+// "@username".
+func (u *User) GetCompleteName() string {
```

Comment:
It is quite confusing to introduce this "common" function into the User model.

For example:

* There is already `GetDisplayName` and `DisplayName`
* Why the `GetCompleteName` has an `@` , is it a general approch .....

(I know the old code is not good, but introducing a new GetCompleteName here makes things more complex ....)